python_ta/cfg/visitor.py

```diff
@@ -68,7 +81,14 @@ def visit_classdef(self, node: nodes.ClassDef) -> None:
             child.accept(self)
 
     def visit_functiondef(self, func: nodes.FunctionDef) -> None:
-        self._current_block.add_statement(func)
+        # If user specifies to only render functions, check if the current function is listed
+        if self._functions_to_render is not None and func.name not in self._functions_to_render:
```

Comment:
It would be good here to also handle methods using their qualified name (`Class.method_name`). From a `FunctionDef` node you can try to get the enclosing scope to see whether it's in a class. The "forbidden IO" checker does something similar.